### DISCUSSION TITLE
beginer help

### DISCUSSION THREAD

**Discussion by CountQuadula** [bezo97/IFSRenderer#26] [Q&A]
hi, 
1) how to create animations? 
2) 4 iterators, please clarify "Set the Center parameters on them using these values: 1, 0, 0, 0,  1, 0, 0, 0,  1 and  1, 1, 1."
X 1, 0, 0, 0,  ?
Y 1, 0, 0, 0, 1 ?
Z1, 1, 1. ?
3) How to add a dding a color palette? (how to make or where to get them)

**Comment by bezo97** [comment]
Hello, here are the answers:
1. Animations are currently in development, it'll take at least a few months to finish. I'll provide a tutorial for animations when it's ready.
2. About the [Sierpinski tutorial](https://github.com/bezo97/IFSRenderer/wiki/Beginner:-Sierpinski-Tetrahedron): You must set the 'Center' parameter on each 4 iterators in the following way:
    - 1st iterator: `1 0 0`
    - 2nd iterator: `0 1 0`
    - 3rd iterator: `0 0 1`
    - 4th iterator: `1 1 1`
 3. A palette editor is also in development. For now, to add color palettes you can create your own using separate programs (such as [ChaosHelper](https://www.deviantart.com/triptychaos/journal/ChaosHelper-1-1-0-Has-Arrived-514357252) or UltraFractal).   
Or just use existing gradient files created by the community - [here's a place to look around](https://www.deviantart.com/fractal-resources/gallery/9825210/gradient-packs).

Hope I could help!
